draft-ietf-mmusic-sctp-sdp.xml

```diff
@@ -638,55 +672,71 @@
+        Once the answerer has sent the answer the answerer:
       </t>
       <t>
-        In the case of TCP/DTLS/SCTP, the answerer follows
-        the procedures in <xref target="RFC4145"/> regarding
-        the establishment/re-establishment of the TCP connection association.
+				<list style="symbols">
+					<t>
+            MUST, in the case of TCP/DTLS/SCTP, follow the procedures in
+            <xref target="RFC4145"/> regarding the establishment/re-establishment
+            of the TCP connection association;
+          </t>
+          <t>
+            MUST follow the procedures in <xref target="I-D.ietf-mmusic-dtls-sdp"/>
+            regarding the establishment/re-establishment of the DTLS association; and
+          </t>
+          <t>
+            MUST, if an SCTP association has yet not been established, or if an existing SCTP
+            association is to be re-established, initiate the establishment
+            of the SCTP association.
+          </t>
+        </list>
+			</t>
+      <t>
+        If the SDP 'sctp-port' attribute in the answer contains a zero attribute
+        value, the answerer MUST NOT establish an SCTP association. If an SCTP
+        association exists, the offerer MUST close it.
       </t>
 			<t>
 				If the answerer does not accept the m- line in the offer, it MUST assign
         a zero port value to the corresponding m- line in the answer, following the
         procedures in <xref target="RFC3264"/>. In addition, the answerer MUST NOT
-        initiate the establishment of an SCTP association, or a DTLS association,
-        associated with the m- line.
+        initiate the establishment of a TCP connection, a DTLS association or a
+        DTLS association associated with the m- line.
 			</t>
 		</section>
 		<section title="Offerer Processing of the SDP Answer">
-			<t>
-        Once the offerer has received the answer, which contains an m- line
-        with a non-zero port value, the offerer MUST, if an SCTP
-        association has yet not been established, or if an existing SCTP
-        association is to be re-established, initiate the establishment
-        of the SCTP association.
+      <t>
+        Once the offerer has received the answer the offerer:
+      </t>
+      <t>
+				<list style="symbols">
+					<t>
+            MUST, in the case of TCP/DTLS/SCTP, follow the procedures in
+            <xref target="RFC4145"/> regarding the establishment/re-establishment
+            of the TCP connection association;
+          </t>
+          <t>
+            MUST follow the procedures in <xref target="I-D.ietf-mmusic-dtls-sdp"/>
+            regarding the establishment/re-establishment of the DTLS association; and
+          </t>
+          <t>
+            MUST, if an SCTP association has yet not been established, or if an existing SCTP
+            association is to be re-established, initiate the establishment
```

Comment:
What does it mean SCTP connection being re-established? Does it mean previous association is closed and new is created?